FirebaseCore/Sources/FIRApp.m

```diff
@@ -606,6 +610,24 @@ + (BOOL)hasSwiftRuntime {
   return hasSwiftRuntime;
 }
 
++ (NSString *)applePlatform {
+  NSString *applePlatform = @"unknown";
+
+#if TARGET_OS_MACCATALYST
```

Comment:
Add a comment summarizing the discussion so we remember this is order-sensitive

Reply:
Added the comment. Is it what you meant?

Reply:
Yep. Thanks!